Dockerfile

```diff
@@ -38,8 +39,43 @@ RUN \
   apt-get remove -y git && \
   apt-get -y autoremove
 
+COPY . ./
+
 USER node
 ENV NODE_ENV=production
 
+ENV MQTT_HOST=localhost \
+    MQTT_PORT=1883 \ 
+    MQTT_QOS=1 \
+    MQTT_RETAIN=false
+
+ENV AMQP_HOST=localhost \
+    AMQP_PORT=5672 \
+    AMQP_EXCHANGE=iota-exchange \
+    AMQP_QUEUE=iotaqueue \
+    AMQP_OPTIONS_DURABLE=true
+
+ENV HTTP_PORT=7896
+
+ENV IOTA_LOGLEVEL=DEBUG \
```

Comment:
I would assume the the default would be a production environment so should the default be `DEBUG` or something less verbose?